app/scripts/lib/app-start.js

```diff
@@ -304,47 +304,53 @@ function (
     },
 
     initializeAuthenticationBroker: function () {
-      if (! this._authenticationBroker) {
-        if (this._isFxDesktop()) {
-          this._authenticationBroker = new FxDesktopAuthenticationBroker({
-            window: this._window,
-            relier: this._relier,
+      function buildBroker() {
```

Comment:
Our unofficial convention is to use `create<X>` when creating a new object. Can you call this function `createBroker`?